weave/flow/eval.py

```diff
@@ -268,6 +280,121 @@ async def evaluate(self, model: Union[Op, Model]) -> dict:
 
         return summary
 
+    def get_evaluate_calls(self) -> CallsIter:
+        """
+        Retrieve all evaluation calls that used this Evaluation object.
+
+        Note that this returns a CallsIter instead of a single call because it's
+        possible to have multiple evaluation calls for a single evaluation (e.g.
+        if you run the same evaluation multiple times).
+
+        Returns:
+            CallsIter: An iterator over Call objects representing evaluation runs.
+
+        Raises:
+            ValueError: If the evaluation has no ref (hasn't been saved/run yet).
+
+        Examples:
+            >>> evaluation = Evaluation(dataset=examples, scorers=[scorer])
+            >>> await evaluation.evaluate(model)  # Run evaluation first
+            >>> calls = evaluation.get_evaluate_calls()
+            >>> for call in calls:
+            ...     print(f"Evaluation run: {call.id} at {call.started_at}")
+        """
+        evaluate_op_name = "Evaluation.evaluate"
+
+        if not self.ref:
+            raise ValueError("Evaluation has no ref, please run the evaluation first!")
+
+        client = require_weave_client()
+        return client.get_calls(
+            query={
+                "$expr": {
+                    "$contains": {
+                        "input": {"$getField": "op_name"},
+                        "substr": {"$literal": evaluate_op_name},
```

Comment:
consider using:
```
eval_op_ref = f"weave:///{client._project_id()}/op/{evaluate_op_name}:*"
query={"$expr": {"$eq": [{"$getField": "op_name"}, {"$literal": eval_op_ref}]}}
```

Reply:
should be more performant and a bit tighter

Reply:
wait actually, this is what we want:

```
eval_op_ref = f"weave:///{client._project_id()}/op/{evaluate_op_name}:*"
client.get_calls(filter={"op_names": [eval_op_ref]})
```

Reply:
Why is this better?

Reply:
Its both more performant in the db and is easier to read as a user

Reply:
specifically, we do a special pre-group by filter for hardcoded conditions (in the filter as opposed to query)